go/vt/sqlparser/ast_funcs.go

```diff
@@ -1354,3 +1354,8 @@ func handleUnaryMinus(expr Expr) Expr {
 	}
 	return &UnaryExpr{Operator: UMinusOp, Expr: expr}
 }
+
+// encodeBytesSQLString encodes the byte string as a SQL string.
+func encodeBytesSQLString(val []byte) string {
```

Comment:
why have this middle-man?

Reply:
ah, saw the uses in `sql.y` now

Reply:
cant have an import in sql.go/sql.y. So have a create a local function in this package that sql.y can then use